Presentation.Web/app/components/it-advice/it-advice-modal-dialog.ts

```diff
@@ -156,7 +181,7 @@
                         $scope.errMessage = "";
                         $scope.startDateErrMessage = "";
                         $scope.curDate = new Date();
-                        if (!moment($scope.startDate, "dd-MM-yyyy").isValid() ||
+                        if (!moment($scope.startDate, allowedDateFormats).isValid() ||
```

Comment:
Vi skal lige have en den til at validere på "strict" format ellers tillader den for meget:
moment($scope.startDate, allowedDateFormats, true)